workflow/rules/common.smk

```diff
@@ -2,8 +2,12 @@ import copy
 from datetime import datetime, timedelta
 import yaml
 import hashlib
+import logging
```

Comment:
I am still not sure we need logging here. The warnings below should at most be debug statements imho and indeed probably can be inferred from the snakemake logs anyway, no?